File: PHP/card.php
<?php
require_one("payment.php");

class card extends payment {
    $id;
    $cardName;
    $cardNumber;
    $cardType;

    public function __construct($id, $cardName, $cardNumber, $cardType){
        parent::__construct($id);
        $this ->$cardName = $cardName;
        $this ->$cardNumber = $cardNumber;
        $this ->$cardType = $cardType;
    }
}


?>

File: python/card.py
from Python.payment import Payment

class Card(Payment):
    id=int
    cardName =str
    cardNumber =str
    cardType =str
    
    def __init__(self,id,cardName,cardNumber,cardType):
        super().__init__(id)
        self.cardName = cardName
        self.cardNumber = cardNumber
        self.cardType = cardType